cmd/object-handlers.go

```diff
@@ -266,15 +266,9 @@ func (api objectAPIHandlers) SelectObjectContentHandler(w http.ResponseWriter, r
 
 	// Executes the query on data-set
 	if err = s3select.Execute(w, s3s); err != nil {
-		logger.LogIf(ctx, err)
-
-	}
-
-	for k, v := range objInfo.UserDefined {
```

Comment:
We need this to be set for audit logging.

Reply:
done